lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module

```diff
@@ -235,18 +250,17 @@ function dosomething_mbp_get_common_campaign_payload(&$payload, $params) {
  *
  * @param string $event_name
  *   Example: "campaign_signup", "password_reset", etc
- * @param string $user_language
- *   The user language setting as defined by dosomething_settings_get_geo_country_code() and
- *   dosomething_global_convert_country_to_language().
+ * @param string $language
+ *   The language setting that the message template should use.
  *
  * @return string
  *   Name of Mandrill template to use for transactional message.
  */
-function dosomething_mbp_get_template_name($event_name, $user_language = NULL) {
-  if (isset($user_langauge)) {
-    $country_code = dosomething_global_convert_language_to_country($user_language);
+function dosomething_mbp_get_template_name($event_name, $language = NULL) {
+  if (isset($langauge)) {
+    $country_code = dosomething_global_convert_language_to_country($language);
     if ($country_code == NULL) {
-      $country_code = 'US';
+      $country_code = 'GL';
```

Comment:
Sounds good. As an alternative, I propose for your consideration `XG`. It is also free for user assignment.